createdisk-library.sh

```diff
@@ -383,3 +383,18 @@ function download_podman() {
       mv podman-remote/windows/podman-${version}/usr/bin/podman.exe  podman-remote/windows
     fi
 }
+
+# As of now sparsify helper is very specific to OCP/OKD kind of bundle where we get the
+# partition for the root label and then mount it with guestfish to cleanup /boot. With
+# microshift vm we are using lvm and guestfish error out during mount
+# mount /dev/sda3 /
+# libguestfs: error: mount: mount exited with status 32: mount: /sysroot: unknown filesystem type 'LVM2_member'
+# There might be other way for guestfish to mount lvm but as of now using a seperate helper is easy.
```

Comment:
I'd aim at the opposite, make use of virt-sparsify in all codepaths. Initially we moved away from it because of issues with encrypted partitions iirc, but I think these are gone. However, now `/` is immutable in the VM image, which is why we use /boot instead. and I'm not sure there's a way around this in virt-sparsify.

Reply:
This is an open question, and therefore needs more time.

> I'd aim at the opposite, make use of virt-sparsify in all codepaths.

Sounds like improvements that can be a new PR; please file an issue and see if this an improvement.

Reply:
Yes, this is an open question and won't be fixed now. However it should  not have taken multiple iterations on this PR to get a clear explanation why code was duplicated.